packages/jest-haste-map/src/module_map.js

```diff
@@ -132,6 +161,14 @@ export default class ModuleMap {
       set,
     );
   }
+
+  static createEmpty() {
```

Comment:
just `create`. It's implied that if you create one without arguments it would be empty.